integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -884,6 +884,12 @@ public static void createDockerRegistrySecret(
     domainMap.put("exposeAdminNodePort", true);
     domainMap.put("adminNodePort", 30700 + number);
     domainMap.put("t3ChannelPort", 30000 + number);
+    if ((System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
+        || (domainMap.containsKey("LB_TYPE")
```

Comment:
LB_TYPE is only environment variable. you can look for "loadBalancer" attribute in domain map.

Reply:
if we chnage here from using LB_TYPE to loadBalancer, we need to change the use of LB_TYPE  in ITStickySession.java

Reply:
Maggie, when you make change here, can you please change ITStickySession.java , line 66 to loadBalancer? That way, ITStickySession tests will still pass with your PR

Reply:
@vanajamukkara @hzhao-github from yesterday's conversation with HuilingI just assumed that now after  ITStickySession change, "LB_TYPE" is the attribute of the domain map since as you pointed out now in the develop branch  Domain.java around line 1352
 this.voyager =
        (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
            || (inputDomainMap.containsKey("LB_TYPE")
                && ((String) inputDomainMap.get("LB_TYPE")).equalsIgnoreCase("VOYAGER"));

Reply:
seems like there was some confusion or misunderstanding ...the condition in Domain.java should be changed to the below...
```
    this.voyager =
        (System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
            || (inputDomainMap.containsKey("loadBalancer")
                && ((String) inputDomainMap.get("loadBalancer")).equalsIgnoreCase("VOYAGER"));
```

Reply:
@vanajamukkara Done. Please review again. Thanks!